pkgs/development/cuda-modules/aliases.nix

```diff
@@ -1,18 +1,13 @@
 # Packges which have been deprecated or removed from cudaPackages
 final: prev:
 let
-  mkRenamed =
-    oldName: newName: newPkg:
-    final.lib.warn "cudaPackages.${oldName} is deprecated, use ${newName} instead" newPkg;
-in
-{
+  # There are currently no aliases , but the infrastructure is desired
+  # preserved. To please the nixpkgs CI check for nixf-tidy, we must not have
+  # unused definitions here. mkRenamed is thus commented out until it is needed
+  # again.
 
-  # Deprecated: an alias kept for compatibility. Consider removing after 24.05
-  autoFixElfFiles = mkRenamed "autoFixElfFiles" "pkgs.autoFixElfFiles" final.pkgs.autoFixElfFiles; # Added 2024-03-30
-  autoAddDriverRunpath =
-    mkRenamed "autoAddDriverRunpath" "pkgs.autoAddDriverRunpath"
-      final.pkgs.autoAddDriverRunpath; # Added 2024-03-30
-  autoAddOpenGLRunpathHook =
-    mkRenamed "autoAddOpenGLRunpathHook" "pkgs.autoAddDriverRunpathHook"
-      final.pkgs.autoAddDriverRunpath; # Added 2024-03-30
-}
+  # mkRenamed =
+  #   oldName: newName: newPkg:
```

Comment:
An arbitrary alternative: one could keep `mkRenamed` and return a `mapAttrs mkRenamed { }`. I'm fine with the comment though

Reply:
Don't want to do this, not arbitrary